<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Update Firmware</string>
  </property>
  <widget class="Backdrop" name="backdrop">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>481</height>
    </rect>
   </property>
  </widget>
  <widget class="HeaderLabel" name="headerlabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Update Firmware</string>
   </property>
  </widget>
  <widget class="Label" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>731</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Step 1: Save your factory-set camera calibration data to an external device.</string>
   </property>
  </widget>
  <widget class="Label" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>731</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Step 2: Insert a USB stick or SD card with the update on it.</string>
   </property>
  </widget>
  <widget class="Label" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>731</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Step 3: Restore the camera calibration from step 1.</string>
   </property>
  </widget>
  <widget class="Label" name="label_4">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>100</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>to</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Button" name="uiSaveCalData">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>241</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Calibration Data</string>
   </property>
  </widget>
  <widget class="Button" name="button_2">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>70</y>
     <width>171</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Safely Remove</string>
   </property>
   <property name="clickMarginLeftSetter" stdset="0">
    <enum>Button::half</enum>
   </property>
  </widget>
  <widget class="ComboBox" name="uiMediaSelect">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>70</y>
     <width>361</width>
     <height>81</height>
    </rect>
   </property>
   <property name="clickMarginRightSetter" stdset="0">
    <enum>ComboBox::half</enum>
   </property>
  </widget>
  <widget class="Button" name="uiApplyUpdate">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>170</y>
     <width>241</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply Software Update</string>
   </property>
  </widget>
  <widget class="Button" name="uiLoadCalData">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>270</y>
     <width>241</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Calibration Data</string>
   </property>
  </widget>
  <widget class="Button" name="uiDone">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>400</y>
     <width>181</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Done</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Backdrop</class>
   <extends>QLabel</extends>
   <header>backdrop</header>
  </customwidget>
  <customwidget>
   <class>Button</class>
   <extends>QPushButton</extends>
   <header>button</header>
  </customwidget>
  <customwidget>
   <class>ComboBox</class>
   <extends>QComboBox</extends>
   <header>combo_box</header>
  </customwidget>
  <customwidget>
   <class>HeaderLabel</class>
   <extends>QLabel</extends>
   <header>header_label</header>
  </customwidget>
  <customwidget>
   <class>Label</class>
   <extends>QLabel</extends>
   <header>label</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
